src/controllers/csi/provisioner/controller_test.go

```diff
@@ -90,14 +110,18 @@ func TestOneAgentProvisioner_Reconcile(t *testing.T) {
 				},
 			),
 			db: metadata.FakeMemoryDB(),
+			gc: gc,
 		}
 		result, err := provisioner.Reconcile(context.TODO(), reconcile.Request{NamespacedName: types.NamespacedName{Name: dynakubeName}})
 
+		gc.AssertNumberOfCalls(t, "Reconcile", 0)
 		assert.NoError(t, err)
```

Comment:
Interesting, I didn't consider it earlier.
But if error occurs in the middle of the test case, there is no point in continuing it because assessments made after something failed are corrupted.

Reply:
Actually it's the opposite, fixed.